packages/jest-config/src/normalize.js

```diff
@@ -547,7 +547,11 @@ export default function normalize(options: InitialOptions, argv: Argv) {
         );
         break;
       case 'testRegex':
-        value = options[key] && replacePathSepForRegex(options[key]);
+        const valueOrEmptyArray = options[key] || [];
+        const valueArray = Array.isArray(valueOrEmptyArray)
+          ? valueOrEmptyArray
+          : [valueOrEmptyArray];
+        value = valueArray.map(replacePathSepForRegex).map(e => new RegExp(e));
```

Comment:
Oh yeah, this is great! Would be great to do all of that in normalize, this is a great start

(not that you should do it now, linking just since it's relevant #6910)

Reply:
converting string to regex may throw (e.g. when a group is unterminated), I think it's worth throwing a meaningful error in such case

Reply:
isn't the error meaningful?

```js
new RegExp('(()');
// throws SyntaxError: Invalid regular expression: /(()/: Unterminated group
```

Reply:
But is it clear that Jest throws that because of wrong configuration?

Reply:
Ah! Gotcha. Can deffo append something that it's coming from jest for `testRegex`